var/spack/repos/builtin/packages/blast-plus/package.py

```diff
@@ -0,0 +1,98 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+import os
+from spack import *
+
+
+class BlastPlus(Package):
+    """BLAST (Basic Local Alignment Search Tool) finds regions of similarity
+    between biological sequences. The program compares nucleotide or protein sequences
+    to sequence databases and calculates the statistical significance"""
+
+    homepage = "http://ncbi.nlm.nih.gov/Blast.cgi"
+
+    variant("static", default=True, description="build static libraries and executables")
+    variant("freetype", default=False, description="build with freetype")
+    variant("gnutls", default=False, description="enable gnutls")
+    variant("hdf5", default=False, description="enable hdf5")
+    variant("jpeg", default=True, description="enable jpeg")
+    variant("libpng", default=True, description="enable libpng")
+    variant("pcre", default=True, description="enable pcre")
+    variant("python", default=True, description="build with python support")
+    variant("debug", default=True, description="Enable debug")
+
+    version('2.6.0', 'c8ce8055b10c4d774d995f88c7cc6225', url="ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/ncbi-blast-2.6.0+-src.tar.gz")
```

Comment:
It is recommended to substitute the  ftp URL with the following https URL. NCBI's HTTPS server seems slow, which takes 60 seconds to begin downloading.

```
url='https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/2.6.0/ncbi-blast-2.6.0+-src.tar.gz'
```